git-extra/git-update-git-for-windows

```diff
@@ -205,6 +203,8 @@ update_git_for_windows () {
 	esac
 
 	echo "Update $latest is available" >&2
+	releases_url=https://api.github.com/repos/git-for-windows/git/releases
+	releases=$(http_get $releases_url/latest) || return
```

Comment:
I guess we do not really need to take care of trying a different proxy for this URL, as `gitforwindows.org` is backed by GitHub Pages?

Reply:
Right. I could not think of a use case where gitforwindows.com should use a different proxy than api.github.com. And refactoring the code to lookup a new proxy each time seems premature.